nuts/nuts_init.py

```diff
@@ -0,0 +1,248 @@
+from typing import Any, Dict, List, Literal
+from pathlib import Path
+
+import typer
+from ruamel.yaml import YAML, CommentedMap
+
+
+app = typer.Typer()
+
+
+def get_host_data(
+    platform: Literal["ios", "nxos_ssh", "eos", "junos", "iosxr"],
+) -> Dict[str, Any]:
+
+    # connection_options = CommentedMap(
+    #     {
+    #         "napalm": CommentedMap({"extras": CommentedMap()}),
+    #         "netmiko": CommentedMap({"extras": CommentedMap()}),
+    #     }
+    # )
+
+    # if platform == "iosxr":
+    #     connection_options["napalm"]["platform"] = "iosxr_netconf"
+    #     option_args = CommentedMap()
+    #     option_args.yaml_add_eol_comment("Workaround for ncclient")
+    #     connection_options["napalm"]["extras"]["optional_args"] = option_args
+
+    return {
+        # "platform": platform,
+        "hostname": "127.0.0.1",
+        "groups": [platform],
+        # "connection_options": connection_options,
+    }
```

Comment:
Does this code provide any value? If no please delete it

Reply:
Added some more logic and rewrote this section